src/Input/DatePickerInput/index.js

```diff
@@ -102,17 +106,25 @@ class DatePickerInput extends PureComponent {
    * @param {Object} textValue - text updated on TextInput
    */
   handleInputChange = textValue => {
-    const { onChange } = this.props;
+    const { minDate, maxDate, onChange } = this.props;
+    onChange(newValue);
+
     this.setState({ textValue });
 
     const [day, month, year] = textValue.split('/').map(v => parseInt(v, 10));
     const newValue = DateTime.fromObject({ year, month, day });
```

Comment:
`const { isValid } = DateTime.fromObject({})`

Reply:
I need the whole object to check if it is within range :d (cf line 121)